src/pybullet_helpers/joint.py

```diff
@@ -0,0 +1,103 @@
+"""Pybullet helper class for joint utilities."""
+from typing import List, NamedTuple, Sequence, Tuple
+
+import numpy as np
+import pybullet as p
+
+from predicators.src.pybullet_helpers.geometry import Pose3D, Quaternion
+
+
+class JointInfo(NamedTuple):
+    """Joint Information from Pybullet."""
+    jointIndex: int
+    jointName: str
+    jointType: int
+    qIndex: int
+    uIndex: int
+    flags: int
+    jointDamping: float
+    jointFriction: float
+    jointLowerLimit: float
+    jointUpperLimit: float
+    jointMaxForce: float
+    jointMaxVelocity: float
+    linkName: str
+    jointAxis: Sequence[float]
+    parentFramePos: Pose3D
+    parentFrameOrn: Quaternion
+    parentIndex: int
+
+    def is_circular(self) -> bool:
```

Comment:
any thoughts about making these properties instead?

Reply:
I did consider it but can't remember why I didn't use it. I think it was so we didn't conflate between the fields from pybullet (in camelcase) and our custom functionality on top of that.

I'm fine with either way

Reply:
Ok Pycharm will indicate these distinctly so I will turn into properties